### PR TITLE
Enable snaps in stable

### PR DESCRIPTION
**Pull request by FrederikBolding**
Enables snaps in stable 🎉 🎉 

Also starts running E2E testing of snaps functionality on stable builds.

### CODE REVIEW

**Review by hmalik88 — DISMISSED** [review]
LGTM

**Review by david0xd — DISMISSED** [review]

**Comment by bowensanders** [comment]
Manual & Automated QA passes ✅

**Review by GuillaumeRx — DISMISSED** [review]

**Review by GuillaumeRx — DISMISSED** [review]

File: .circleci/config.yml [hunk 1]
```diff
@@ -883,11 +963,11 @@ jobs:
       - store_test_results:
           path: test/test-results/e2e.xml
 
-  test-e2e-chrome-snaps:
+  test-e2e-chrome-snaps-flask:
     executor: node-browsers
     parallelism: 4
     steps:
-      - run: *shallow-git-clone
+      - checkout
```
Comment (Gudahtt) [inline .circleci/config.yml:970]: Could we use a shallow clone here as well?

Comment (FrederikBolding) [reply .circleci/config.yml:970]: For sure! I must have missed that when rebasing!

**Review by Gudahtt — APPROVED** [review]
LGTM!

**Review by david0xd — APPROVED** [review]

**Review by hmalik88 — APPROVED** [review]

**Pull request merged**